### PR TITLE
Chore: Add a quick start guide

### PR DESCRIPTION
**Pull request by Blokje5**
## Description

 

Added a basic quick start guide, showcases one of the use cases of chproxy: Spreading INSERTs accross a ClickHouse cluster.

I've also included an examples directory so other contributors can contribute more examples of chproxy usage.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

### Checklist

- [ ] Linter passes correctly
- [ ] Add tests which fail without the change (if possible)
- [ ] All tests passing
- [ ] Extended the README / documentation, if necessary

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: docs/content/en/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+Optionally, to verify that the cluster is running properly, execute the following from a new terminal shell:
+
+```console
+docker exec -it quick_start_clickhouse_1 clickhouse-client --query "SELECT COUNT(*) FROM tutorial.views"
+```
+
+The ClickHouse server exposes an [HTTP interface](https://clickhouse.com/docs/en/interfaces/http/) on port 8123. Chproxy uses the HTTP interface to submit queries to Clickhouse.
+
+The ClickHouse servers are already setup with a Distributed Table on each node, as described in [Spreading INSERTs](/use-cases/spread-insert).
```
Comment (mga-chka) [inline docs/content/en/getting_started.md:30]: I'm not sure if this relative link will work (to bad, I can't see to generated webdoc from render from your PR to test the link)

**Review by mga-chka — APPROVED** [review]
good tutorial.
Might be worth adding a filesystem cache later in the tuto (since caching is one of the main features of chproxy)

File: docs/content/en/getting_started.md (hunk 1 above)
Comment (Blokje5) [reply docs/content/en/getting_started.md:30]: I used this syntax as I saw more links using this relative syntax. But maybe those are the broken links mentioned in #231.

**Comment by Blokje5** [comment]
> good tutorial. Might be worth adding a filesystem cache later in the tuto (since caching is one of the main features of chproxy)

I considered adding Caching as well. I didn't want to explain to much in a single tutorial, but it might be worth it including it anyway, and it follows up nicely on the story of spreading and reducing load.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Changelog
 menuTitle: Changelog
 category: Guides
-position: 103
+position: 104
```
Comment (mga-chka) [inline CHANGELOG.md:5]: why do you modify the position to 104 if no files is linked to 103? Did you planned to have your getting started at 103 and not 105, which is also the position of the faq?

Comment (mga-chka) [reply CHANGELOG.md:5]: @Blokje5 (I'm pinging just in case you didn't see this msg)

Comment (Blokje5) [reply CHANGELOG.md:5]: Yep, that was my intent indeed. I wanted the getting started guide on position 103.

**Comment by Blokje5** [comment]
@mga-chka I've also included a section on query caching. PTAL.

**Review by mga-chka — APPROVED** [review]

**Pull request merged**